source/get-started.txt

```diff
@@ -218,13 +218,17 @@ your operating system:
    .. tab:: MacOS
       :tabid: install-mac
 
-      You can download the {+driver-short+} on MacOS by using the `Homebrew
-      package manager <https://brew.sh/>`__. To do so, run the following command in
-      your shell:
+      To use the {+driver-short+} on MacOS, install the driver from the
+      source code. For installation instructions, see the :ref:`c-install-from-source`
```

Comment:
"install the driver from the source code" -> "build the C Driver libraries from source".